package.json

```diff
@@ -45,52 +45,50 @@
     "redis": "^2.7.1",
     "reflect-metadata": "^0.1.10",
     "sprintf-js": "^1.0.3",
-    "url-join": "^1.1.0",
+    "url-join": "^2.0.2",
     "winston": "^2.3.1",
-    "winston-sentry": "^0.1.5",
+    "winston-sentry": "^0.2.1",
     "yamljs": "^0.2.10"
   },
   "devDependencies": {
+    "@types/auth0-js": "^8.6.0",
+    "@types/bluebird": "^3.5.5",
```

Comment:
Yes, I thought it was missing from the @types/knex's package.json, but apparently when I ran `npm rm @types/bluebird` I should've run `npm install` afterwards. Removed now.